language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/hover/HoverUtil.java

```diff
@@ -120,6 +136,52 @@ private static Hover getHoverForSymbol(Symbol symbol, HoverContext context) {
         }
     }
 
+    /**
+     * Get hover for expression nodes. Note that we are supplying hover for a selected set of expressions.
+     *
+     * @param context  Context
+     * @param exprNode Expression node
+     * @return Hover
+     */
+    private static Hover getHoverForExpression(HoverContext context, Node exprNode) {
+        switch (exprNode.kind()) {
```

Comment:
missing default case

Reply:
I did not add a default case to keep the code simple. If we add a default case, we will have to return the default hover object in the default case and at the end of the method both. Wdyt?

Reply:
We can leave the default case empty.(`default:`) I thought it is best to add it